main.go

```diff
@@ -457,6 +474,20 @@ func fetchDistKey(c *cli.Context) error {
 	return nil
 }
 
+func controlShare(c *cli.Context) error {
+	port := c.String("port")
+	if port == "" {
+		port = net.DefaultControlPort
+	}
+	client := net.NewControlClient(port)
+	share, err := client.Share()
+	if err != nil {
+		slog.Fatalf("drand: could not request the share: %s", err)
+	}
+	log.Printf("\n{\n\tprivate share, %s\n}", share.String())
```

Comment:
I would prefer if you just output the share in plain JSON, to follow the UNIX philosophy where each tools can be combined with others tools, if we stick to "standard" output :)